src/libs/ReportUtils.ts

```diff
@@ -993,6 +993,13 @@ Onyx.connect({
     callback: (value) => (allPolicies = value),
 });
 
+let allPolicyDrafts: OnyxCollection<Policy>;
+Onyx.connectWithoutView({
+    key: ONYXKEYS.COLLECTION.POLICY_DRAFTS,
+    waitForCollectionCallback: true,
+    callback: (value) => (allPolicyDrafts = value),
+});
+
```

Comment:
I think we still need to avoid using Onyx.connect. Is this the only way to handle `getReportOrDraftReport`. Can we just rely on the passing params instead?